### DISCUSSION TITLE
How to remove the ring from the `@tailwindcss/forms`'s checkbox?

### DISCUSSION THREAD

**Discussion by frederikhors** [tailwindlabs/tailwindcss#10981] [Help] [answered]
Please, help me: I'm loosing my mind.

I'm using `@tailwindcss/forms` and what I want is to remove the white border outside the checkbox and have the outline as on a simple button:

![image](https://user-images.githubusercontent.com/41120635/231013720-bcc7c99e-bc57-495e-a1dd-8e22064dc1b0.png)

I don't understand why the tailwind team is adding the white border (a ring maybe?) on the checkbox and how to remove that ring!

Thanks in advance!

https://play.tailwindcss.com/KchzkiVx0s

**Comment by adamwathan** [comment, marked as answer]
Hey! That's the ring offset you're seeing:

https://play.tailwindcss.com/rRRFic7CoR

Hoping to simplify all this crap in a future release now that we can start relying on `outline` instead of rings for things with all major browsers respecting border radius for outlines.

**Reply by collimarco** [reply]
Tailwind Forms and this ring seem very unintuitive and different from the rest of Tailwind. Usually you add styles in Tailwind, while here you struggle to remove the default styling

**Reply by rangelojc** [reply]
Has this ever been removed? This styling kills my sanity every couple of months
